src/darsia/corrections/shape/curvature.py

```diff
@@ -122,6 +122,7 @@ def __init__(
                 self.cache_path = Path(
                     self.config.get("cache", "./cache/curvature_transformation.npy")
                 )
+                self.cache_path.parents[0].mkdir(parents=True, exist_ok=True)
```

Comment:
Good that you added this! I manually made this directory when I ran the code earlier..